README.md

```diff
@@ -27,7 +27,7 @@ File `app-config.json` must be provided. If no configuration file is supplied th
 ```json
 {
     "encryptionKey": "encryptionKey",
-    "apiUrl": "https://play.dhis2.org/2.30/api",
+    "baseUrl": "https://play.dhis2.org/2.30",
```

Comment:
If no baseUrl is defined, it continues anyway and tries a `GET undefined/api/schemas`. Maybe we should validate the values from the json?